src/Singular.jl

```diff
@@ -11,6 +11,7 @@ import AbstractAlgebra
 using Markdown
 using Nemo
 using Pkg
+using lib4ti2_jll
```

Comment:
Perhaps a test could be added to ensure it is working?

Reply:
You mean a test that checks that the Singular-4ti2 interface actually works? Yes, that would definitely be a good idea. Perhaps @fieker has some example code handy (from the notebook by @wdecker ) ? Otherwise, @tthsqe12 should be able to figure something out from the manual?

Reply:
The notebook is running - but has its own interface to 4ti2 inside. I am reluctant to change this...(as it is currently working)

Reply:
No problem, @tthsqe12 added his own test code in the meantime. 

So as long as this doesn't actively break that notebook, and if you have no other concerns, we should just merge this.